front_end/sdk/DartSupport.js

```diff
@@ -1,6 +1,8 @@
+// Functions specific to Dart support.
```

Comment:
nit: clarify the comment. Seems like this a module specific to Dart support with everything added under the Dart namespace.

Reply:
Done.